app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -669,6 +676,10 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
+    private fun configureVoiceSearch() {
```

Comment:
This function isn't configuring anything, maybe rename to `logVoiceSearchAvailability`?